examples/sonar_text_demo.ipynb

```diff
@@ -9,19 +9,50 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 3,
```

Comment:
why this files existing were modified ? you may want to create a dedicated demos notebook for hf use case )